easybuild/tools/systemtools.py

```diff
@@ -338,6 +338,9 @@ def get_os_version():
                     ('2.6.32', '_SP1'),
                     ('3.0', '_SP2'),
```

Comment:
Do you mind updating these too, while we're at it, since it's unaware of SP3 or SP4, and `3.0` doesn't necessarily mean SLES 11 SP2.

according to https://wiki.microfocus.com/index.php/Kernel_versions, things got more complicated though...

The kernel versions for SLES 11 SP2 now range from 3.0.13-0.27.1 to 3.0.101-0.7.17.1, while SP3 ranges from 3.0.76-0.11.1 to 3.0.101-0.47.67.2, and SP4 is 3.0.101-63.1.

So, it seems the logic we were using got broken. :-(
So, I'm not sure how to handle this...

Maybe like this (since order matters):

```
"11": [
    ('2.6.27', ''),
    ('2.6.32', '_SP1'),
    ('3.0.101-63', '_SP4'),
    # not 100% correct, since early SP3 had 3.0.76 - 3.0.93, but close enough?
    ('3.0.101', '_SP3'),
    # SP2 kernel versions range from 3.0.13 - 3.0.101
    ('3.0', '_SP2'),
],
```